admin/src/pages/dashboard/Table.tsx

```diff
@@ -0,0 +1,185 @@
+// @flow
+import React, { useState } from "react";
+import { UserOutlined } from "@ant-design/icons";
+import { Button, Table } from "antd";
+import type { ColumnsType } from "antd/es/table";
+import Avatar from "antd/es/avatar/avatar";
+import Style from "./table.module.css";
+import { ArrowDownOutlined } from "@ant-design/icons";
+interface DataType {
+  key: React.Key;
+  author: string;
+  title: string;
+  address: string;
+  number: string;
+}
+
+const columns: ColumnsType<DataType> = [
```

Comment:
create folder `tableColumns`, move `columns` to `tableColumns` and rename to `docsColumns`